client/package.json

```diff
@@ -9,7 +9,8 @@
     "test:watch": "ng test",
     "lint:fix": "ng lint --fix",
     "lint": "ng lint",
-    "setup": "node setup.js"
+    "setup": "node setup.js",
+    "generate-graphql": "graphql-codegen --config graphql-codegen.yml"
```

Comment:
Nit: This is process is called introspection. Might want to call it that as it the same in other projects.